OsmAnd-java/src/main/java/net/osmand/GPXUtilities.java

```diff
@@ -1657,8 +1660,14 @@ public int getNonEmptySegmentsCount() {
 		}
 
 		public QuadRect getRect() {
-			double left = 0, right = 0;
-			double top = 0, bottom = 0;
+			return getRectAroundPoint(0, 0);
+		}
+
+		public QuadRect getRectAroundPoint(double lat, double lon) {
```

Comment:
2 methods are not needed one is enough

getBounds(double defaultMissingLat, double defaultMissingLon)